UnleashClient/features/Feature.py

```diff
@@ -58,6 +61,15 @@ def increment_stats(self, result: bool) -> None:
         else:
             self.no_count += 1
 
+    def count_variant(self, variant_name: str) -> None:
```

Comment:
This looks like an internal method that should only be called by the SDK itself, so it might be better off being a non public method. That being said, we don't do that anywhere else in the file so I won't fuss too much if you disagree

Reply:
Yeah, that's totally fair. I agree that that might be better. Does python have a way to make anything non-public? The closest might be using the dunderscore, i.e:

```suggestion
    def __count_variant(self, variant_name: str) -> None:
```

Reply:
Can't edit the suggestion directly so I made a new one!